web/src/graphQL.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { useQuery } from "@apollo/client";
+
+import { gql } from "../src/__generated__/gql";
+
+const GET_PROJECTS = gql(`
+    query getProjects {
+        myProjects {
+            id
+        }
+    }
+`);
+export const GraphQL: React.FunctionComponent<{}> = () => {
```

Comment:
Is this a test component, or a ProjectSelector?

Reply:
Just a test component. It's still a draft PR, just wanted to get something working. Once all the schema setup (+linting etc) have been setup, I'll delete this component and start making bigger changes.